tests/test_bear_deploy.py

```diff
@@ -31,7 +31,7 @@
     'config': {
      'NOInst': {
       'growth_commod': 'bears',
-      'growth_rate': 0.05,
```

Comment:
I think these input parameters have to be updated to match the updated input parameters for NOInst